docs/designs/data_transform.md

```diff
@@ -1,4 +1,84 @@
-# Data Tranform Design Doc
+# Data Analysis and Tranformation
+
+## The Problem
+
+We want to add data analysis, in addition to data transformation, into the end-to-end machine learning pipeline generated by SQLFlow.
```

Comment:
In this design doc, we also want to add some built-in functions for transformation. 
For example `standardize` is mapped to a python function.

Reply:
I am afraid that it is too early to tell such details and distracts readers from the most important points.